src/app/reactUtils.ts

```diff
@@ -51,10 +51,12 @@ export const onFocusInOrOutHandler = (
 };
 
 export const useFocusLost = (ref: React.RefObject<HTMLElement>, isEnabled: boolean, cb: () => void) => {
+  // eslint-disable-next-line react-hooks/exhaustive-deps
   React.useEffect(onFocusInOrOutHandler(ref, isEnabled, cb, 'focusout'), [ref, isEnabled]);
```

Comment:
is this because of the callback?

Reply:
This is because `onFocusInOrOutHandler` is a separate function

Reply:
it should be correct to add that to deps, if it is re-defined it might have new scope that is necessary, you can wrap it in React.useCallback to reduce reloading

Reply:
or if you really want to ignore its scope changes you could wrap it in a ref

Reply:
I can add `cb` to dependencies but the issue will still exist. It is the same as for `onClickOutside`:
https://github.com/openstax/rex-web/pull/1156#discussion_r625741872